SVGKit-iOS.xcodeproj/project.pbxproj

```diff
@@ -2801,6 +2809,7 @@
 				ENABLE_TESTABILITY = YES;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
```

Comment:
Add `Carthage` search path before the embed version framework. So carthage user will firstlly hit the local `CocoaLumberjack` version. (CocoaLumberjack supports Carthage).

For manually install user, this does not cause issue, and they will hit the `(PROJECT_DIR)/3rd-party-frameworks/CocoaLumberjack-2.2.0/iOS` one.